### DISCUSSION TITLE
Windows Explorer `Open with Geany` potential bug

### DISCUSSION THREAD

**Discussion by ghost** [geany/geany#3755] [General]
Related to #3754. But it doesn't take many files to trigger. Sometimes, only two files. It's more likely happen with files started with the dot in their names, e.g: the files on MSYS2's home directory like `.bashrc`, `.profile`,... But, this is not always the case. Sometimes, it triggered when I'm opening a bunch of `.cs` files. Note: the number of opened files is no where as much as on #3754.

I guess it's because I'm opening files too fast that triggered the bug.

**Comment by elextr** [comment]
Again, what are you doing, we can't look over your shoulder, how are you "too fast" .

When reporting bugs (on any project) you have to describe it well enough that others can reproduce the problem, don't assume that someone can guess, if the problem hasn't been reported before then you are doing something different to everyone else, so you need to describe exactly what you are doing and versions on Geany and Windows.

**Comment by ghost** [comment]
Or better yet, I will just accept that Geany is only for a handful of files and let JEdit handle a large amount of files. I have described everything I have done in the best way I can, in detail. How could you expect more?

**Comment by elextr** [comment]
Well, I still do not actually know what you did.  Reporting issues needs a step by step recipe to reproduce including the version of all software involved.

Here on Linux Mint 20 Cinnamon I selected all the files in the Geany `src` directory in the file manager (thats 110 files according to it) and selected `Open with -> Geany` and they all opened in Geany 2.0, and all in the same instance.  

Is that what you did on Windows?  And what version of Windows? And what version of Geany?  And how many did you try to open?
